### PR TITLE
feat(backingImage) BackingImage page refactor

### PR DESCRIPTION
**Pull request by smallteeths**
wangsiye/longhorn-ui:4d2b788

Signed-off-by: Siye Wang <siye@rancher.com>

### CODE REVIEW

**Comment by innobead** [comment]
Pending for backend merge.

File: src/components/Replica/Replica.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ class Replica extends React.Component {
       if (rebuildError) {
         return <div style={{ display: 'flex', justifyContent: 'flex-end', marginLeft: '20px' }}><Tooltip title={rebuildError}><Icon style={{ color: '#faad14' }} type="warning" /></Tooltip></div>
       } else {
-        return progress === 0 || progress === 100 ? '' : <Tooltip title={`${state === 'restore' ? 'Restoring' : 'Deleting Snapshot'} ${progress}%`}><Progress status={state === 'restore' ? 'success' : 'exception'} percent={progress} showInfo={false} /></Tooltip>
+        return progress === 0 || progress === 100 ? '' : <Tooltip title={`${state === 'restore' ? 'Restoring' : 'Deleteing Snapshot'} ${progress}%`}><Progress status={state === 'restore' ? 'success' : 'exception'} percent={progress} showInfo={false} /></Tooltip>
```
Comment (innobead) [inline src/components/Replica/Replica.js:82]: Deleting

**Review by innobead — APPROVED** [review]
LGTM, but some typo (Deleteing needed to fix)

**Comment by smallteeths** [comment]
> LGTM，但有些拼写错误（需要删除以修复）

Thank you for your patience in checking.

**Pull request merged**